verticaltabs.js

```diff
@@ -706,7 +706,12 @@ VerticalTabs.prototype = {
     };
     window.addEventListener('aftercustomization', afterListener);
 
-    window.addEventListener('resize', this.resizeTabs.bind(this), false);
+    window.addEventListener('resize', () => {
+      this.resizeTabs();
+      if (this.pinnedWidth > document.width / 2) {
+        document.documentElement.style.setProperty('--pinned-width', `${Math.min(this.pinnedWidth, document.width / 2)}px`);
```

Comment:
So, if we're doing the `Math.min`, then we don't need to check whether it's smaller or not, because the `Math.min` will handle that for us…  And thinking about it, we should probably change it to the `min` [on startup](https://github.com/bwinton/TabCenter/blob/b6e5d611e857afa24c6ed90fe944e2be2b5c12ee/verticaltabs.js#L440), too…